driver-core/src/main/com/mongodb/client/model/vault/EncryptOptions.java

```diff
@@ -153,28 +140,28 @@ public Long getContentionFactor() {
     /**
      * The QueryType.
      *
-     * <p>It is an error to set queryType when algorithm is not "Indexed".</p>
+     * <p>It is an error to set queryType when the algorithm is not "Indexed".</p>
      *
      * @param queryType the query type
      * @return this
      * @since 4.7
      */
     @Beta(Beta.Reason.SERVER)
-    public EncryptOptions queryType(@Nullable final QueryType queryType) {
+    public EncryptOptions queryType(@Nullable final String queryType) {
```

Comment:
Consider adding Javadoc that notes the currently-supported query type.  So that users don't have to look far to find it.  (We do this in getAlgorithm method, though maybe easier to find if it were on the setters as well)